platform/accessibility/vsp-accessibility-rules-of-engagement.md

```diff
@@ -0,0 +1,93 @@
+# VSP Accessibility Design & Development Rules of Engagement 
+
+**Please note:**  <br/>This review process will continue to evolve. Please contact Trevor in Slack with suggestions, questions, or improvements.
+
+## Accessibility Design Checkpoints
+
+Accessibility specialists are included in VSP design reviews, as outlined in the [VSP Design Rules of Engagement](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/design/working-with-platform-design-team.md#whentorequest). Recommendations will be based on [best practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md) for inclusive design and development.
+
+For small projects, use your judgment for what levels of support and feedback you need. Generally, a pre-usability testing design review or a design QA review will be appropriate for this type of project, but you are the expert!
```

Comment:
Regarding:
> For small projects, use your judgment for what levels of support and feedback you need.

I worry teams lack the judgment to know what levels of support and feedback they need. Regarding accessibility and inclusive design, especially, the lack of awareness could impact the project's timeline at inopportune moments — and launch an inaccessible outcome for our VA population. 

Consider suggesting that they post in #vfs-platform-support to arrange a consult on what may be appropriate.

Reply:
I hesitate to give them the free reign that they are the expert when it comes to accessibility consult.